environment.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
```

Comment:
Just reuse the one from CI: https://github.com/openmm/openmm-torch/blob/master/devtools/conda-envs/build-ubuntu-18.04.yml